### DISCUSSION TITLE
nikidrillion’s assets

### DISCUSSION THREAD

**Discussion by nikidrillion** [fishfolk/jumpy#566] [Assets]
https://user-images.githubusercontent.com/102508189/212915773-2ffec38f-3aa4-4e72-905c-6988dc78f28d.mp4
1. Coral with bubles 
2. New seaweed (i can change colors if purple doesn't fit or do it less bright)
3. Seashell 1&2 (it's like crate)
4. Sword-fish (it's like sword, but i think we can use this model as rapier)
![Coral with bubles 1](https://user-images.githubusercontent.com/102508189/212918434-1537595c-b2e0-463f-883e-c38d611be0d5.png)
![new seaweed 1](https://user-images.githubusercontent.com/102508189/212918439-da4a5dc5-56a4-4d81-ad4c-0c5884bb7b5e.png)
![seashell 1 ](https://user-images.githubusercontent.com/102508189/212918445-5928f070-1688-4a43-818d-dad9ffbf171a.png)
![seashell 2 ](https://user-images.githubusercontent.com/102508189/212918450-8c428aaa-b6ec-4929-9da9-a45437208579.png)
![new sword 1 ](https://user-images.githubusercontent.com/102508189/212918442-387c8fbd-2989-494b-9d3f-b5ca1a3ef6c3.png)

**Comment by nikidrillion** [comment]
Improved stomp boots (original by @zTecna) sprite: 
1. add spikes to make it look more dangerous
2. add outline

Preview:
![new spikes boots](https://user-images.githubusercontent.com/102508189/213127940-47eb725e-c685-406b-b15f-8bcbb9fe3243.gif)
Icon: 
![spikes boots](https://user-images.githubusercontent.com/102508189/213127977-610711fb-de5c-4d9d-b61a-d876c0b73e2c.png)
Sprite sheet: ![new spikes boots](https://user-images.githubusercontent.com/102508189/213127964-390aff80-6b35-4004-b735-a3f8b120d109.png)
Sprite sheet with Sharky: ![new spikes boots with sharky](https://user-images.githubusercontent.com/102508189/213128023-a9ad1ed3-e0e3-4a70-a958-d7fd09618b69.png)

**Comment by nikidrillion** [comment]
I made separate hands for each character.
![Preview](https://user-images.githubusercontent.com/102508189/213872362-ee7786a4-fc94-4bb9-a8a3-5d94f9d9678c.png)

![Sharky without hand](https://user-images.githubusercontent.com/102508189/213871767-24e2b5f4-fd35-41c9-86c1-fbe1dcf2ea9e.png)
![Lionfishy without hand](https://user-images.githubusercontent.com/102508189/213871809-7f8ab479-7107-4f13-a301-2acffe47b5e7.png)
![Pescy without hand](https://user-images.githubusercontent.com/102508189/213871779-9aa92351-9bce-4240-85e4-e3a657cb87ea.png)
![Orcy without hand](https://user-images.githubusercontent.com/102508189/213871780-152f80ca-5d82-4986-9bc9-eb5e74ccd78b.png)
![Fishy without hand](https://user-images.githubusercontent.com/102508189/213871788-07ff8e74-fde6-44c0-829e-402818bfb184.png)
![Catty without hand ](https://user-images.githubusercontent.com/102508189/213871789-1d4bc761-6582-4012-90bf-df6934d68385.png)

**Reply by nikidrillion** [reply]
Now I think I'll do a crouching pose for each character.
I will leave the old sprite and add new ones on the right, like this.
![image](https://user-images.githubusercontent.com/102508189/213872797-1c9049b7-cba9-4460-a91f-64b99571e841.png)

**Comment by nikidrillion** [comment]
I made a new background!

Preview:
![background  3 underwater](https://user-images.githubusercontent.com/102508189/216641437-2cce49d9-47d1-4d63-901f-6c0a6642e0d9.png)
light:
![light](https://user-images.githubusercontent.com/102508189/216641571-8e9d137f-8c1b-4633-8554-41cf1ff8e246.png)
frame:
![frame](https://user-images.githubusercontent.com/102508189/216641590-52644df3-359c-4e8c-b719-1bebbd28a30a.png)
background_03:
![background_03](https://user-images.githubusercontent.com/102508189/216641608-9e8d1afd-2ddd-4612-a059-4be2f15ff334.png)
background_02:
![background_02](https://user-images.githubusercontent.com/102508189/216641620-eef13dd6-4f7b-4483-801f-9b78bb99f8bb.png)
background_01:
![background_01](https://user-images.githubusercontent.com/102508189/216642132-750aa94f-b631-4d0a-bca9-a1293925ea18.png)
background_01 + submarine:
![background_01 + submarine](https://user-images.githubusercontent.com/102508189/216642148-06fdf81a-f945-4341-ac02-012d635d72d1.png)

**Reply by zicklag** [reply]
Nice! Good job.

**Comment by nikidrillion** [comment]
Crouching pose + Sliding.

![Catty ](https://user-images.githubusercontent.com/102508189/218778947-0fa0f840-806e-46a6-9b7f-100f3bc26580.png)
![Fishy ](https://user-images.githubusercontent.com/102508189/218778958-b7c6f81d-161a-432a-83d8-ea1f24554947.png)
![Lionfishy ](https://user-images.githubusercontent.com/102508189/218778969-0d8c2d80-d450-43a5-8ea8-bcc2300c485c.png)
![Orcy ](https://user-images.githubusercontent.com/102508189/218778984-fff23c69-3ef4-464a-b677-ddb31d2fa5bc.png)
![Pescy ](https://user-images.githubusercontent.com/102508189/218778993-45b324df-5949-4e18-bc2e-9fc9250d3b65.png)
![Sharky ](https://user-images.githubusercontent.com/102508189/218779004-3ff237d4-1cc9-4148-974a-9217f1062eb2.png)
